src/core/utils.ts

```diff
@@ -277,6 +277,9 @@ export function getPotionID(uri: string, resourceURI: string): PotionID {
 export function findPotionResource(uri: string, resources: PotionResources): {resourceURI: string, resource: typeof Item} | undefined {
     if (isString(uri)) {
         const entry = Object.entries(resources)
+            .sort((a, b) => {
```

Comment:
You don't need the curly braces, you can `([a], [b]) => a.length - b.length`.

Reply:
Ok - I will change it.